lib/enigma.rb

```diff
@@ -1,8 +1,40 @@
+require 'date'
 class Enigma
+  def rand_key
+    rand(99999)
+  end
 
-  def pad(number)
-    pad_num = number.to_s
+  def zero_pad(key)
+    pad_num = key.to_s
     pad_num.insert(0, "0") until pad_num.size == 5
     pad_num
   end
```

Comment:
"zero_pad" will take our key, or random number, and ensure that it is a 5 digit number by adding zeros to it until it is 5 digits long.